app/view_models/workarea/storefront/google_product_feed_sku_view_model.rb

```diff
@@ -13,6 +25,22 @@ def condition
         'new'
       end
 
+      def image_url
+        return unless image.present?
+        ProductImageUrl.product_image_url(image, GoogleProductFeed.image_size)
+      end
+
+      def image
+        @image ||= sku_image || images.primary
```

Comment:
You could reuse the standard `ProductViewModel` code here, and it would give the advantage of including any customizations on the project as well as not needing additional code. Try this:
```
Storefront::ProductViewModel.wrap(product, current_sku: model.sku).primary_image
```

Reply:
Sorry about that, I've had a busy week and was going to make some time this weekend.

With your recommendation, are you suggesting replacing the whole of line 34 or just the images.primary condition with Storefront::ProductViewModel.wrap(product, current_sku: model.sku).primary_image

Reply:
No problem, looks like you got it.